webapp/channels/src/components/notify_counts/notify_counts.tsx

```diff
@@ -4,15 +4,14 @@
 import React from 'react';
 
 import type {BasicUnreadMeta} from 'mattermost-redux/selectors/entities/channels';
-type Props = BasicUnreadMeta;
 
-export default class NotifyCounts extends React.PureComponent<Props> {
-    render() {
-        if (this.props.unreadMentionCount) {
-            return <span className='badge badge-notify'>{this.props.unreadMentionCount}</span>;
-        } else if (this.props.isUnread) {
-            return <span className='badge badge-notify'>{'•'}</span>;
-        }
-        return null;
+const NotifyCounts = ({unreadMentionCount, isUnread}: BasicUnreadMeta) => {
+    if (unreadMentionCount) {
+        return <span className='badge badge-notify'>{unreadMentionCount}</span>;
+    } else if (isUnread) {
+        return <span className='badge badge-notify'>{'•'}</span>;
     }
-}
+    return null;
+};
+
+export default NotifyCounts;
```

Comment:
I'm missing here the `memo` because it was a `PureComponent`.
```suggestion
export default React.memo(NotifyCounts);
```

Reply:
Removing memo because it fails the tests

Reply:
I didn't get any solution to pass the tests, remove memo helps to pass testcase

Reply:
The `React.memo` is needed, the tests needs to be changed for that. Let me help there, but removing is not the solution.

Reply:
Yes, it would be helpful

Reply:
Fixed, take a look to the last commit. It was tricky because enzime wasn't playing nice with `React.memo` there. Anyway, we can end up getting rid of `enzime` and use `react-testing-library`. But this fix just works.